Prepare for 3.2.5 (#252)
Iterable/iterable-android-sdk: CHANGELOG.md

#### Fixed
- nothing yet

## [3.2.5](https://github.com/Iterable/iterable-android-sdk/releases/tag/3.2.5)

> #### Added

Reviewer: ```suggestion
#### Changed
```

> - `app_name` was removed from published strings and replaced with plain string values in test manifests.

Reviewer: I think this should be in 'Changed' rather than 'Added'.

Author: oh yes

#### Fixed
- nothing yet

## [3.2.5](https://github.com/Iterable/iterable-android-sdk/releases/tag/3.2.5)
#### Added

> - Introducing authToken. `setUserId` and `setEmail` methods now takes optional `authToken` parameter

Author: @vbabenkoru 
Do you think we should include this only when backend is ready? and everything is tested?

Reviewer: We should make the methods private and ship 3.2.5 without this note. Once everything is ready for JWT, make them public again, ship 3.2.6 and add a link to the JWT docs in the changelog.

Author: Should I make those methods private in this PR? OR should I have it seperate and rebase this one?

Reviewer: I think ideally a separate PR, but here is also ok.

> - `app_name` is set to `IterableAPI` in strings resource file

Reviewer: ```suggestion
- `app_name` was removed from published strings and replaced with plain string values in test manifests.
```

#### Fixed

> - Fixed an issue where trackInAppClicks were not being registered after displaying an inApp.

Reviewer: ```suggestion
- Fixed an issue where in-app click events were not being registered after displaying an in-app message.
```

> - Fixed crash issues by handling null cases caused in `IterableInAppFragmentHTMLNotification` class

Reviewer: ```suggestion
- Fixed NullPointerExceptions in `IterableInAppFragmentHTMLNotification` that could occur in some cases when the activity is destroyed and recreated.
```
